cmake/FetchDependencies.cmake

```diff
@@ -143,6 +151,9 @@ if(NC_BUILD_NCCONVERT)
     )
 
     FetchContent_MakeAvailable(assimp)
+
+    disable_warnings_for_headers(assimp)
```

Comment:
Can these not go above in the block where all the other `disable_warnings` function calls are?